modules/module-postgres-storage/README.md

```diff
@@ -0,0 +1,88 @@
+# Postgres Sync Bucket Storage
+
+This module provides a `BucketStorageProvider` which uses a Postgres database for persistence.
+
+## Configuration
+
+Postgres storage can be enabled by selecting the appropriate storage `type` and providing connection details for a Postgres server.
+
+The storage connection configuration extends the configuration for a Postgres replication source, thus it accepts and supports the same configurations fields.
```

Comment:
"extends the configuration" feels weird here - can we just say it uses the same configuration fields?

Reply:
👍 Updated the wording.